fastlane/lib/fastlane/version.rb

```diff
@@ -1,5 +1,5 @@
 module Fastlane
-  VERSION = '2.210.1'.freeze
+  VERSION = '2.210.2'.freeze
```

Comment:
Please revert this change as it shouldn't be part of a PR 😊 this gets updated during deployment processes